src/evaluators/loader.py

```diff
@@ -17,3 +28,30 @@ def load_infmax_model(config: dict[str, Any], random_seed: int, nb_seeds: int) -
         return load_model({"model": config})
     else:
         return lambda x: "dupa"  # TODO: add here GT loader
+
+
+def if_stochastic(infmax_model: Callable) -> bool:
+    match infmax_model.__class__.__name__:
```

Comment:
[2]
I suggest 
```
if infmax_model.__class__.__name__ in {"MultiNode2VecKMeans", "MultiNode2VecKMeansAuto"}:
    return True
return False
```
(or even an if-else statement) is more Pythonic.

General note: i feel match-case statements are a bit overused. Their main use case is structural pattern matching - a case that does not occur here. Changing every 2-branch if into match-case statement isn't really Pythonic.  (it's not like C++ switch statement, which does have an advantage over consecutive ifs)

Reply:
Thanks for pointing this out. I've applied your remark